### DISCUSSION TITLE
Search does not work

### DISCUSSION THREAD

**Discussion by lushnone** [typesnippet/antd-phone-input#39] [General]
Hello, thank you very much for releasing antd-phone-input. I am currently using antd version 5.9.x, and in this version, the enableSearch feature does not work properly. It's unable to search for phone number area codes.

**Comment by ArtyomVancyan** [comment]
Hi @brucetod, thanks for the feedback.

I have tested with the latest version of both `antd` (5.9.0) and `antd-phone-input` (0.1.8) and it worked for me.

1. Click on the flag to open the dropdown menu with the countries list.
2. Search `383` on the search input and see two matching results.

![Screenshot from 2023-09-12 16-59-56](https://github.com/ArtyomVancyan/antd-phone-input/assets/44609997/810ee12e-e0b0-4f43-bbea-2c1e3a5f0c30)

My **package.json** contains the following and run with node v19.9.0 and npm v9.6.3
```json
"dependencies": {
  "antd": "^5.9.0",
  "antd-phone-input": "^0.1.8",
  "react": "^18.2.0",
  "react-dom": "^18.2.0",
  "react-scripts": "^5.0.1"
}
```

Could you please provide the code you are facing the issue with?
